### ISSUE TITLE
object has no attribute 'value'

### ISSUE THREAD

**Issue by tuxfamily** [msztolcman/smtpc#2]
Hello.

Just installed smtpc (on mac OS Catalina / python 3.9), and I got this error:

```shell
$ smtpc send --host 10.0.4.20:1025 --body-type html --subject 'Some html email' --body-html 'Some <b>HTML</b> message body' --from not-funny@smtpc.net --to receiver@smtpc.net
Traceback (most recent call last):
  File "/usr/local/bin/smtpc", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.9/site-packages/smtpc/cli.py", line 560, in main
    handler.handle()
  File "/usr/local/lib/python3.9/site-packages/smtpc/cli.py", line 432, in handle
    self._handle()
  File "/usr/local/lib/python3.9/site-packages/smtpc/cli.py", line 470, in _handle
    message_builder = message.Builder(
  File "/usr/local/lib/python3.9/site-packages/smtpc/message.py", line 96, in __init__
    self.body_type = guess_content_type(self.body_type.value, self.body_plain, self.body_html)
AttributeError: 'str' object has no attribute 'value'
```

I try different options, but similar error:

```shell
$ smtpc send --host 10.0.4.20:1025 --from sendria@example.com --to you@example.com --subject 'Welcome!' --body 'Welcome to Sendria!'
Traceback (most recent call last):
  File "/usr/local/bin/smtpc", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.9/site-packages/smtpc/cli.py", line 560, in main
    handler.handle()
  File "/usr/local/lib/python3.9/site-packages/smtpc/cli.py", line 432, in handle
    self._handle()
  File "/usr/local/lib/python3.9/site-packages/smtpc/cli.py", line 470, in _handle
    message_builder = message.Builder(
  File "/usr/local/lib/python3.9/site-packages/smtpc/message.py", line 96, in __init__
    self.body_type = guess_content_type(self.body_type.value, self.body_plain, self.body_html)
AttributeError: 'NoneType' object has no attribute 'value'
```

Here is my setup:

```shell
$ python3 --version
Python 3.9.2
```

```shell
$ pip3 install 'smtpc[extended]'
Requirement already satisfied: smtpc[extended] in /usr/local/lib/python3.9/site-packages (0.8.0)
Requirement already satisfied: toml in /usr/local/lib/python3.9/site-packages (from smtpc[extended]) (0.10.2)
Requirement already satisfied: structlog in /usr/local/lib/python3.9/site-packages (from smtpc[extended]) (21.1.0)
Requirement already satisfied: fileperms in /usr/local/lib/python3.9/site-packages (from smtpc[extended]) (1.0.4)
Requirement already satisfied: colorama in /usr/local/lib/python3.9/site-packages (from smtpc[extended]) (0.4.4)
Requirement already satisfied: Jinja2 in /usr/local/lib/python3.9/site-packages (from smtpc[extended]) (2.11.3)
Requirement already satisfied: MarkupSafe>=0.23 in /usr/local/lib/python3.9/site-packages (from Jinja2->smtpc[extended]) (1.1.1)
```

Any suggestions?

Thank you.

**Comment by msztolcman** [comment]
Hello @tuxfamily,

It's unfortunately just a plain bug, fixed now in 0.8.1.
Please update your `smtpc` version to newest version:
`python3 -m pip install -U 'smtpc[extended]'`

Should works better :)

Thanks for your bug report!

**Comment by tuxfamily** [comment]
It works 👍 
Thank you very for this very fast fix (and for sendria and smtpc 😁)
